### PR TITLE
[Port] Better Lizards 2

### PR DESCRIPTION
**Pull request by PuroSlavKing**
# Описание PR

Taken from: https://github.com/space-syndicate/space-station-14-next/pull/68

4 новых еблета, анимация для сигма-хвоста и версия наспинных шипов без шипов на хвосте.

## Медиа

image

## Тип PR

- [ ] Feature
- [ ] Fix
- [x] Tweak
- [ ] Balance
- [ ] Refactor
- [x] Port
- [ ] Translate
- [ ] Resprite

**Изменения**

## Release Notes

- **New Features**
	- Introduced a variety of new reptilian markings, including heads, shouts, and tails, enhancing customization options for characters.
	- Added new texture states for reptilian models, providing more visual variety.
  
- **Bug Fixes**
	- Updated state names for clarity and consistency across various resources.

- **Documentation**
	- Updated copyright information in texture resources.

### CODE REVIEW

File: Resources/Prototypes/_Backmen/Entities/Mobs/Customization/Markings/reptilian.yml [hunk 1]
```diff
@@ -9,16 +18,64 @@
...
+- type: marking
+  id: SnakeLargeTailAnimated
+  bodyPart: Tail
+  markingCategory: Tail
+  speciesRestriction: []
+  sprites:
+    - sprite: Backmen/Mobs/Customization/reptilian_64x32.rsi
+      state: snake_large_wagging
+    - sprite: Backmen/Mobs/Customization/reptilian_64x32.rsi
+      state: snake_large_wagging_overlay
```
[the remark this answers is not in the document]
Comment (PuroSlavKing) [reply Resources/Prototypes/_Backmen/Entities/Mobs/Customization/Markings/reptilian.yml:78]: this is an animated version of snake large tail, it shouldn't have species restrictions as it's just animation

**Comment by nomad0260** [comment]
ужас какой

**Comment by PuroSlavKing** [comment]
> ужас какой
> 

:(

**Review by nomad0260 — APPROVED** [review]
выглядит убедительно

**Pull request merged**